docs/reference/backups.md

```diff
@@ -70,3 +70,105 @@ As of version `0.0.21` invoke-kubesae, has a utility command `utils.scale-app` t
     ```shell
     kubectl -n mywebapp-production exec -it deploy/app -- python manage.py createsuperuser
     ```
+
+## Adding Hosting Services Backups to a Project
+
+This documentation is meant to clarify intent and process for developers who may want to add hosting 
+services backup to their projects. [Site for the ansible role that does all of this](https://github.
+com/caktus/ansible-role-k8s-hosting-services/).
+
+NOTE: This documentation is primarily meant as reference. If your team doesn't want to set this up on your 
+project yourself, you can ask the Sysadmin team to set up the backups on your project.
+
+There are two components to setting up hosting services backups to a project.
+
+1. The variable setup in your `host_vars/<environment_host>.yaml` and `k8s.yaml` files
+2. Deploying the service into your projects cluster. (i.e. creating the service namespace and setting up the requested 
+   cron jobs)
+
+### Variable setup
+
+For the majority of setups, you will need only a few variables.
+
+#### Access Variables
+
+These variables allow the Caktus Hosting services AWS Role to do work in your project.
+
+Currently, this should be same on all projects.
+
+```yaml
+k8s_hosting_services_aws_access_key: <PROVIDED_BY_SYSADMIN>
+k8s_hosting_services_aws_secret_access_key: <PROVIDED_BY_SYSADMIN>
+```
+
+#### Identification Variables
+
+These variables identify key things about how you want your projects backups to be stored.
+
+```yaml
+# This could be any string that you want to use to help identify the backup folder
+k8s_hosting_services_project_name: "<PROJECT_NAME>"
+# This will be the folder that the backups are stored in.
+k8s_hosting_services_namespace: "{{ k8s_hosting_services_project_name }}-hosting-services"
+# The database that the service will be backing up. This will most likely already be set with
+# this particular variable name.
+k8s_hosting_services_database_url: "{{ env_database_url }}"
+# This is the name of the S3 bucket. The `k8s_hosting_services_namespace` folder will be in here.
+k8s_hosting_services_backup_base_bucket: investigator-portal-production-backups
```

Comment:
Should this be more generic?

Reply:
Sorry, Scott I can't tell exactly what you are referencing. Line 117?

Reply:
Nevermind. Derp. I see now. Yes it should thank you.